src/mail-app/mail/view/MailView.ts

```diff
@@ -564,6 +574,27 @@ export class MailView extends BaseTopLevelView implements TopLevelView<MailViewA
 		showMoveMailsDropdown(locator.mailboxModel, mailLocator.mailModel, getMoveMailBounds(), selectedMails)
 	}
 
+	private labels() {
+		const mailList = this.mailViewModel.listModel
+		if (mailList == null) {
+			return
+		}
+
+		const selectedMails = mailList.getSelectedAsArray()
+		if (isEmpty(selectedMails)) {
+			return
+		}
+
+		const popup = new LabelsPopup(
+			this.dom!,
```

Comment:
the element that we pass to popup is just needed to return the focus back after the popup is closed. I think we could pass `document.activeElement` into it. There's a chance that the element will be gone after we apply labels but I think it would be better in most cases.

additionally, we should not use typescript's unwrap feature (exclamation mark) in most cases, we should do checks or `assertNotNull()` depending on the case

Reply:
dom is replaced with document.activeElement